src/components/ApplicationContext/ApplicationContext.tsx

```diff
@@ -133,6 +151,18 @@ export const ApplicationProvider: React.FC = () => {
     }
   }
 
+  const saveData = () => {
+    const data: SavedApplication = {
+      contact: contactFormData,
+      demographic: demographicFormData,
+      shortAnswer: shortAnswerFormData,
+      priorExperience: priorExperienceFormData,
+      connected: connectedFormData,
+      MLH: mlhFormData,
+    }
+    store("application", data, undefined, user?.email)
```

Comment:
Can we also put a timer on this so it doesn't stay indefinitely in our user's localstorage.

Reply:
Yep! A month would be `60 * 60 * 24 * 30` seconds right? Just wanted to make sure...

Also, the timeout is just for our logic, it'll still stay in the localStorage if the user doesn't visit our site after a month. So we should probably clear this after the user submits, and/or after the application deadline.

Reply:
I think that's a good call and for our onSubmit we can handle that